Tests/GitHubContents.tests.ps1

```diff
@@ -0,0 +1,151 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+<#
+.Synopsis
+   Tests for GitHubContents.ps1 module
+#>
+
+# This is common test code setup logic for all Pester test files
+$moduleRootPath = Split-Path -Path $PSScriptRoot -Parent
+. (Join-Path -Path $moduleRootPath -ChildPath 'Tests\Common.ps1')
+
+try {
+    # Define Script-scoped, readonly, hidden variables.
+    @{
+        repoGuid       = [Guid]::NewGuid().Guid
```

Comment:
Can you capitalize these (PascalCase) (and their uses) to indice that it's a constant vs a local var?

Reply:
Sure thing, i was just following the convention from the `GitHubComments.tests.ps1`, in there these items are camelCase such as `defaultIssueTitle`. Or are you referring specifically to `repoGuid` since that is not a constant?